src/plasmapy/analysis/swept_langmuir/helpers.py

```diff
@@ -145,3 +155,82 @@ def check_sweep(  # noqa: C901, PLR0912
         current = current.value
 
     return voltage, current
+
+
+def sort_sweep_arrays(
```

Comment:
Do we need this as part of the public API? 🤔

Reply:
@rocco8773 said that a lot of the arguments to other functions need to be sorted, so yes it does!